magicctapipe/scripts/lst1_magic/lst_m1_m2_cloud_correction.py

```diff
@@ -227,21 +233,35 @@ def process_telescope_data(input_file, config, tel_id, camgeom, focal_eff):
         cleanmask = dl1_images["image_mask"][index_img]
         peak_time = dl1_images["peak_time"][index_img]
         image /= trans_pixels
-        corr_image = image
+        corr_image = image.copy()
         corr_image[~cleanmask] = 0
 
         hillas_params = hillas_parameters(camgeom, corr_image)
-        timing_params = timing_parameters(camgeom, corr_image, peak_time, hillas_params)
+        timing_params = timing_parameters(
+            camgeom[cleanmask],
+            corr_image[cleanmask],
+            peak_time[cleanmask],
+            hillas_params,
+        )
         leakage_params = leakage_parameters(camgeom, corr_image, cleanmask)
-        conc_params = concentration_parameters(camgeom, corr_image, hillas_params)
+        if assigned_tel_ids["LST-1"] == tel_id:
```

Comment:
please add a small comment before this line: "For LST-1 we compute concentration fro the cleaned image and for MAGIC from the full image to reproduce the current behaviour in the standard code.